examples/cryptocurrency/src/lib.rs

```diff
@@ -210,20 +251,29 @@ impl Transaction for TxTransfer {
     /// [`TxCreateWallet`]: struct.TxCreateWallet.html
     fn execute(&self, view: &mut Fork) -> ExecutionResult {
```

Comment:
Shouldn't we also check for `sender == receiver`?

Reply:
I'm not sure: this seems logically wrong, but technically gives no problem, right? We probably don't want to add unnecessary details to the tutorial.

@slowli What do you think?

Reply:
If I understand correctly, transactions with `sender == receiver` don't pass the `verify()` test, so during transaction execution, it is guaranteed that these two aren't equal.